packages/svg/src/icons/Circle/CircleIcon.test.tsx

```diff
@@ -8,7 +8,7 @@ describe('CircleIcon', () => {
 
   it('applies width and height', () => {
```

Comment:
sorry , didnt see other files . were the tests handwritten

Reply:
rather than having tests for every icon we should have a single test that will walkthrough all icons and check if they are rendering properly .  otherwise these might not be mantainable

Reply:
that can work, i was just making changes for now on the first couple of directories before applying this to the rest of them. after this pr is merged, ill revised to moved the testing to a main directory

Reply:
yup  . we will only  need one test for all the icons .

Reply:
@aabidsofi19 it's moved now